src/card.css

```diff
@@ -0,0 +1,92 @@
+.card--content {
+  width: 200px;
+}
```

Comment:
It shouldn't be a modifier of this block. It should be an element of another block. For example, you can add `page` class to the body and make `page__card` for it

Reply:
fixed